api/src/main/java/com/epam/pipeline/manager/git/GitlabClient.java

```diff
@@ -331,6 +326,15 @@ public GitRepositoryEntry createProjectHook(String hookUrl) throws GitClientExce
         return addProjectHook(projectId, hookUrl);
     }
 
+    public GitProject updateProjectName(final String currentName, final String newName) throws GitClientException{
+        final String normalizedNewName = convertPipeNameToProject(newName);
```

Comment:
use explicit call here `GitUtils.convertPipeNameToProject` instead of static import